src/count.js

```diff
@@ -20,6 +20,9 @@ function count(ast) {
     case 'keyframes':
     case 'import':
     case 'supports':
+    case 'charset':
+    case 'namespace':
+    case 'page':
```

Comment:
Hi Matt,

Not sure what you meant if this update *'can be counted against you in IE 9'* by Microsoft, did you mean if we ignore these at-rules they will break the css?

For the documentation I can only find this one, https://msdn.microsoft.com/en-us/library/aa358815(v=vs.85).aspx and they seem to support those 3 selectors.

> ...mask and obscure another bug...

Which bug are you referring to? Do you have a reference? Or you meant it may introduce new bugs?

For additional testing, I tested this simple code in IE 6, 7, 8, 9 and they all worked fine:

**blessed1.css**:
```css
@charset "UTF-8"; /* Test @charset, Test @charset as first line */
@import url(http://fonts.googleapis.com/css?family=Roboto:300,400,700|Roboto+Slab:300,400|Lato:100,300,400,900);

@page :first {
  margin: 1in;
}

@namespace url(http://www.w3.org/1999/xhtml); /* Namespace for XHTML */
@namespace svg url(http://www.w3.org/2000/svg); /* Namespace for SVG embedded in XHTML */

body {
  background-color: red !important;
}
```

**blessed.css**
```css
@import url('app-ie-blessed1.css?z=453');

body {
  color: green;
}
```

expected: Red background with Green text